torch/__init__.py

```diff
@@ -1648,5 +1648,20 @@ def _sparse_coo_tensor_unsafe(*args, **kwargs):
     return torch.sparse_coo_tensor(*args, **kwargs)
 
 
+if sys.executable != 'torch_deploy':
+    class _TritonLibrary(object):
+        lib = torch.library.Library("triton", "DEF")
+        ops_table = {}
```

Comment:
When `torch_diplay`, the library has a duplicate value in it's dict and hence throws an error. Is it possible to circumvent this somehow?